bin/verilator

```diff
@@ -4733,6 +4733,36 @@ Warns that an instance has a pin which is not connected to another signal.
 Disabled by default as this is a code style warning; it will simulate
 correctly.
 
+=item PINNOTFOUND
+
+Warns that an instance port or Parameter was not found in the module being
+instanciated. Please note that Verilator raises these errors also on instances
+that should be disabled by generate/if/endgenerate constructs:
+
+    module a;
+    localparam A=1;
+    generate
+    if (A==0)
+    begin
+    b b_inst1 (.x(1'b0)); // nonexistent port
+    b #(.PX(1'b0)) b_inst2 (); // nonexistent parameter
+    end
+    endgenerate
+    endmodule
+
+    module b;
+    endmodule
+
+In the example above, b is instanciated with a port named x, but module b has
+no such port at all. In the next line, b is instantiated again, this tim with
+a nonexistent parameter PX. Technically, this code is in error because of
+this, but other commercial tools ignore it because the generate/if condition
+is false, and so module b in this example is not instanciated.
```

Comment:
```suggestion
In the example above, b is instantiated with a port named x, but module b has
no such port. In the next line, b is instantiated again with
a nonexistent parameter PX. Technically, this code is incorrect because of
this, but other may tools ignore it because module b is not instantiated
due to the generate/if condition being false.
```